### PR TITLE
drop deprecated katello modules

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,6 @@ This is a list of modules currently in the repository (please add to the list if
 
  * katello_sync: sync Katello repositories and products
  * katello_upload: upload files, rpms, etc. to repositories. Note, rpms & files are idempotent.
- * katello_content_view_publish: publish Katello content views
```
Comment (mdellweg) [inline README.md]: Can you add katello_content_view_version to the list above? I think it is missing.

Comment (evgeni) [reply README.md]: I actually considered dropping the list completely in favor of https://theforeman.org/plugins/foreman-ansible-modules

Comment (mdellweg) [reply README.md]: ... which is autogenerated. :1st_place_medal: 
For the riddance!

Comment (evgeni) [reply README.md]: I'd do that in a separate PR tho.

Comment (mdellweg) [reply README.md]: Why? Do you want to collect PR-Points?

**Pull request merged**